posthog/api/test/test_decide.py

```diff
@@ -629,23 +629,23 @@ def test_feature_flags_v3_consistent_flags_with_numeric_distinct_ids(self):
             self.assertTrue(response.json()["featureFlags"]["beta-feature"])
             self.assertTrue(response.json()["featureFlags"]["default-flag"])
 
-        with self.assertNumQueries(9):
+        with self.assertNumQueries(11):
```

Comment:
these two extra queries are just adding and releasing the savepoint as a consequence of the new transaction.atomic() block